src/content/api/loaders.md

```diff
@@ -471,3 +509,96 @@ Hacky access to the Compiler object of webpack.
 ### `this._module`
 
 Hacky access to the Module object being loaded.
+
+
+## Error Reporting
+
+There are some ways to throw out errors from inside a loader:
```

Comment:
this sounds abstract, please be more specific. e.g. `Here are the ways to throw an error from inside a loader:...`